library_generation/generate_pr_description.py

```diff
@@ -156,24 +156,13 @@ def __filter_qualified_commit(paths: Dict[str, str], commit: Commit) -> (Commit,
 
 
 def __combine_commit_messages(
-    current_commit: Commit,
-    baseline_commit: Commit,
     commits: Dict[Commit, str],
     is_monorepo: bool,
+    repo_level_message: list[str],
 ) -> str:
-    messages = [
```

Comment:
This is pretty useful to handwritten library owners, they even proposed to enhance it to a diff. We don't have to change anything, but let's still keep it if there is a googleapis_commitish change.

Reply:
I restored this change.